backend/conferences/models.py

```diff
@@ -15,6 +15,17 @@ class Conference(TimeFramedModel, TimeStampedModel):
     topics = models.ManyToManyField('conferences.Topic', verbose_name=_('topics'))
     languages = models.ManyToManyField('languages.Language', verbose_name=_('languages'))
     audience_levels = models.ManyToManyField('conferences.AudienceLevel', verbose_name=_('audience levels'))
+    submission_types = models.ManyToManyField('submissions.SubmissionType', verbose_name=_('submission types'))
+
+    @property
+    def is_cfp_open(self):
+        try:
+            cfp_deadline = self.deadlines.get(type=Deadline.TYPES.cfp)
+
+            now = timezone.now()
+            return cfp_deadline.start <= now <= cfp_deadline.end
```

Comment:
I like this, I was thinking that we should probably add the timezone to the conference, so that we can have conf in multiple timezones :)

Reply:
Added